### PR TITLE
Human attributes v2

### PR DESCRIPTION
**Pull request by ddc67cd**

### CODE REVIEW

File: lunavl/sdk/estimators/body_estimators/body_attributes.py [hunk 1]
```diff
@@ -429,6 +366,193 @@ def asDict(self) -> List[str]:
...
+        "isGrey": "gray",
+        "isOrange": "orange",
+        "isPurple": "purple",
+        "isRed": "red",
+        "isYellow": "yellow",
+        "isPink": "pink",
+        "isBrown": "brown",
+        "isBeige": "beige",
+        "isKhaki": "khaki",
+        "isMulticolored": "multicolored",
```
Comment (matemax) [inline lunavl/sdk/estimators/body_estimators/body_attributes.py]: Researchers give a promise that will be used only two palettes.   `OutwearColorEnum` exists now already. Seems like it needs to rename it  and use for LowerGarment and OutwearColor. Also need create palette for HeadWear, shoes and other.

File: lunavl/sdk/estimators/body_estimators/body_attributes.py [hunk 2]
```diff
@@ -429,6 +366,193 @@ def asDict(self) -> List[str]:
...
+    }
+    _unknown = "undefined"
+    _types = {
+        LowerBodyClothing.Unknown: "undefined",
+        LowerBodyClothing.Skirt: "skirt",
+        LowerBodyClothing.Pants: "trousers",
+        LowerBodyClothing.Shorts: "shorts",
+    }
```
Comment (matemax) [inline lunavl/sdk/estimators/body_estimators/body_attributes.py:562]: need enum

File: lunavl/sdk/estimators/body_estimators/body_attributes.py [hunk 3]
```diff
@@ -479,15 +608,20 @@ def __init__(self, coreEstimation):
         else:
             self.sleeve = Sleeve(coreEstimation.sleeve_opt.value())
 
+        self.lowerGarment = LowerGarment(coreEstimation.lowerBodyClothing_opt)
+        self.shoes = Shoes(coreEstimation.shoeColor_opt)
```
Comment (matemax) [inline lunavl/sdk/estimators/body_estimators/body_attributes.py]: implicit that self.shoes may be None.

Comment (matemax) [reply lunavl/sdk/estimators/body_estimators/body_attributes.py]: may be force typing.

**Review by matemax — APPROVED** [review]

File: lunavl/sdk/estimators/body_estimators/body_attributes.py (hunk 3 above)
Comment (ddc67cd) [reply lunavl/sdk/estimators/body_estimators/body_attributes.py]: +

File: lunavl/sdk/estimators/body_estimators/body_attributes.py (hunk 2 above)
Comment (ddc67cd) [reply lunavl/sdk/estimators/body_estimators/body_attributes.py:562]: added

File: lunavl/sdk/estimators/body_estimators/body_attributes.py (hunk 1 above)
Comment (ddc67cd) [reply lunavl/sdk/estimators/body_estimators/body_attributes.py]: unified

**Pull request merged**